goosebit/updater/controller/v1/routes.py

```diff
@@ -22,11 +22,9 @@
 
 
 @router.get("/{dev_id}")
-async def polling(request: Request, dev_id: str, updater: DeviceManager = Depends(get_update_manager)):
+async def polling(request: Request, dev_id: str, device: Device = Depends(get_device)):
```

Comment:
Can remove `dev_id` and use `device.uuid` here instead.

Reply:
Fixed.